shttps/LuaServer.h

```diff
@@ -162,6 +162,8 @@ namespace shttps {
 
```

Comment:
I think I can guess what these methods do, but it would be helpful to add Doxygen comments to make it clear.